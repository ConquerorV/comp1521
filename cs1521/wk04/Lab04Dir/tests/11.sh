echo 5 | spim -file fac3.s
