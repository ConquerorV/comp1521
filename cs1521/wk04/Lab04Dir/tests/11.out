Loaded: /home/cs1521/lib/spim/exceptions.s
n  = n! = 120
